build-scripts/components/cluster-agent/version.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-echo "main"
+echo "MK-698/traefik-compatible-config"
```

Comment:
TODO: revert after https://github.com/canonical/microk8s-cluster-agent/pull/8/files is merged